docs/cel2/notices/day-1-partners.md

```diff
@@ -9,7 +9,7 @@ Below is the list of confirmed Day 1 partners:
 
 - Alchemy
 - All that Node
-- Ankr
+- [Ankr](https://rpc.ankr.com/celo)
```

Comment:
This gives me
```
{"error":"message: WebSocket is disabled, json-rpc code: -32092, rest code: 403"}
```
which probably is not what we want here.

Reply:
Ignore this, this is meant to be the direct RPC access, not a link to the RPC provider's page. So this is ok.
```
> cast block-number -r https://rpc.ankr.com/celo
31073602
```